source/_integrations/energyzero.markdown

```diff
@@ -63,6 +63,7 @@ Fetches the gas prices.
 
 | Service data attribute | Optional | Description | Example |
 | ---------------------- | -------- | ----------- | --------|
+| `config_entry` | no | Config entry to use. Can be found using the Services tab in the Developer Tools and switching to YAML. | 1b4a46c6cba0677bbfb5a8c53e8618b0
```

Comment:
My advice would be not to place the config entry ID explanation in the table, because tables are difficult to read.

![image](https://github.com/home-assistant/home-assistant.io/assets/20448157/744c707a-6a62-45cd-ad68-83f073802c1a)